code/game/objects/items/weapons/material/swords.dm

```diff
@@ -149,4 +149,61 @@
 	icon_state = "amohdan_sword"
 	item_state = "amohdan_sword"
 	contained_sprite = 1
-	slot_flags = SLOT_BELT
\ No newline at end of file
+	slot_flags = SLOT_BELT
+
+// improvised sword
+/obj/item/weapon/material/sword/improvised_sword
+	name = "selfmade sword"
+	desc = "A crudely made, rough looking sword. Still appears to be quite deadly."
+	use_material_name = FALSE
+	icon = 'icons/obj/improvised.dmi'
+	icon_state = "sword"
+	item_state = "sword"
+	force_divisor = 0.3 // 18 damage
+	contained_sprite = 1
+	slot_flags = SLOT_BELT
+

```

Comment:
```suggestion

/obj/item/weapon/material/sword/improvised_sword/Initialize()
	. = ..()
	force_divisor *= (material.weight / 20)
```
Heavier weapons should be better at cutting due to their inertia. This also means the base damage for a steel sword would be the same, 18.